answer_rocket/skill.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from mypy_extensions import Arg
```

Comment:
think this is an auto-import oops? I believe you want Arg from sgqlc. likely what broke the tests

Reply:
Yep, that was it - I just fixed that so hopefully it should build now - thanks!